python/ray/autoscaler/azure/defaults.yaml

```diff
@@ -86,6 +87,7 @@ worker_nodes:
 file_mounts: {
 #    "/path1/on/remote/machine": "/path1/on/local/machine",
 #    "/path2/on/remote/machine": "/path2/on/local/machine",
+     "/home/ubuntu/.ssh/id_rsa.pub": "~/.ssh/id_rsa.pub"
```

Comment:
Is there a reason why the SSH Key is in the file mounts?

Reply:
Changes to NodeLauncher read the contents of public ssh file (in hash_launch_conf) on the head node. Previously this file was not actually added to the head node which caused a failure.

If there's a better way to sync / create this file on the head node that would be preferable, this just seemed like the easiest solution.

Reply:
Oh! I totally missed that these were the `public keys`! I don't know of a better way to do this.